### PR TITLE
Glabs Traffic Driver: QA

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Fixes [bugs bugs bugs](https://docs.google.com/document/d/1DfX_8GA-_C5jzH2AdeGHQTpoO-K9lY47DmNZ4i9HZ2E/edit?ts=58106c9d)

cc @ScottPainterGNM @JonNorman

### CODE REVIEW

File: src/glabs-native-traffic-driver/web/index.scss [hunk 1]
```diff
@@ -38,9 +40,14 @@ $brand-colour: #69d1ca;
 }
 
 .creative__standfirst {
+    display: none;
     margin: 8px 10px 5px;
 }
 
+.creative__standfirst--yep {
```
Comment (regiskuckaertz) [inline src/glabs-native-traffic-driver/web/index.scss]: I changed it to `yes` in a separate PR 😆

File: src/_shared/scss/_core.scss [hunk 2]
```diff
@@ -187,6 +187,20 @@ $creative-max-width: 1920px;
     }
 }
 
+.popup-toggle {
+    position: absolute;
+    visibility: hidden;
+    pointer-events: none;
+
+    &:checked ~ .popup {
```
Comment (regiskuckaertz) [inline src/_shared/scss/_core.scss]: @Calanthe thanks for the tip!

**Comment by regiskuckaertz** [comment]
[Glabs Traffic Driver: QA fixes](https://trello.com/c/sL2qhVxX/175-glabs-traffic-driver-qa-fixes)

File: src/glabs-native-traffic-driver/test.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
 	"ArticleImageAlternateText": "Some Text",
 	"ArticleHeaderText": "",
 	"ArticleText": "",
+	"ShowText": "nope",
```
Comment (JonNorman) [inline src/glabs-native-traffic-driver/test.json]: Do we need to change this to "no"? 😢

**Pull request merged**